src/main/java/org/openeid/cdoc4j/PKCS11TokenParams.java

```diff
@@ -5,13 +5,19 @@
     private String pkcs11Path;
     private String pin;
     private int slot;
+    private String label = null;
```

Comment:
No explicit need to set it to null - When created all instance/class variables will be initialized with a default value which in this case is null.